Compiled from "UnexpectedInvocationException.scala"
public class org.mockito.exceptions.misusing.UnexpectedInvocationException extends org.mockito.exceptions.base.MockitoException {
  public static java.lang.Throwable $lessinit$greater$default$2();
    Code:
       0: getstatic     #16                 // Field org/mockito/exceptions/misusing/UnexpectedInvocationException$.MODULE$:Lorg/mockito/exceptions/misusing/UnexpectedInvocationException$;
       3: invokevirtual #18                 // Method org/mockito/exceptions/misusing/UnexpectedInvocationException$.$lessinit$greater$default$2:()Ljava/lang/Throwable;
       6: areturn

  public org.mockito.exceptions.misusing.UnexpectedInvocationException(java.lang.String, java.lang.Throwable);
    Code:
       0: aload_0
       1: aload_1
       2: aload_2
       3: invokespecial #22                 // Method org/mockito/exceptions/base/MockitoException."<init>":(Ljava/lang/String;Ljava/lang/Throwable;)V
       6: return
}
